lib/render/VolumeCellTraversal.cpp

```diff
@@ -412,6 +412,26 @@ void VolumeCellTraversal::SetUniforms(const ShaderProgram *s) const
 
 float VolumeCellTraversal::GuestimateFastModeSpeedupFactor() const { return 2; }
 
+int VolumeCellTraversal::CheckHardwareSupport(const Grid *grid) const
+{
+    int ret = VolumeRegular::CheckHardwareSupport(grid);
+    if (ret < 0)
+        return ret;
+
+    long freeKB = oglGetFreeMemory();
```

Comment:
The function oglGetFreeMemory() returns an int. So declare freeKB as int?

Reply:
freeKB will be promoted to a long later on in the code, so it's ok to just do it here.